include/wasm/WasmEnvironment.h

```diff
@@ -9,14 +10,22 @@ class WasmEnvironment
   public:
     WasmEnvironment();
 
-    void addEnv(const std::string& val);
+    void addEnv(const std::string& key, const std::string& value);
+
+    std::string addFromSystemEnv(const std::string& key,
+                                 const std::string& fallback);
 
     uint32_t getEnvCount();
+
     uint32_t getEnvBufferSize();
 
-    const std::vector<std::string>& getVars();
+    std::vector<std::string> getVars();
+
+    std::string getEnv(const std::string& key);
+
+    void print();
```

Comment:
Note to self: all this stuff may not be needed if we don't actually need to set `LC_CTYPES`